cmd/completion.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// completionCmd represents the completion command
+var completionCmd = &cobra.Command{
```

Comment:
what is this command for?

Reply:
This is the completion suggestion command. Idea is like when User type `crda [tab][tab]`  he should see acceptable subcommands..

Reply:
Should we really need a command completion? We hardly have 2 commands now.

Reply:
it should be okay either way.